main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strconv"
+	"syscall"
+)
+
+const ProcessesLimit = "10"
+
+func main() {
+    if len(os.Args) < 2 {
+        panic("Few arguments")
+    }
+
```

Comment:
this is not a very user friendly `CLI` interface. Instead you either use the minimalist `flag` library for simple `cli` or more sophisticated libraries like https://github.com/urfave/cli for more complex/modern clis.